### DISCUSSION TITLE
Anyone else have an intermittent issue where the app fails to grant privileges?

### DISCUSSION THREAD

**Discussion by andrewcrook** [SAP/macOS-enterprise-privileges#157] [General]
I occasionally have an issue where gaining privileges doesn’t work.

All the windows go off screen, I get the dialogue , click request, enter password and the windows don't reappear until I click on the edge of the screen. Everything comes back, however, the privileges app doesn’t change state.

- A couple of times this was when using a password and had to open laptop and use touchid. 
- A few times nothing worked and  I had to restart the machine.

I haven’t raised an issue yet because I feel that I don't have enough information is it possible to enable logging or a debug mode?

App version  2.0.0 (1815)
macOS 15.2 (24C101)
MacBook Pro M1 13” + external Apple Extended Keyboard + external moniter LG4K

**Comment by mthielemann** [comment]
@andrewcrook  Please try with the latest beta [Privileges_2.1.0b1925.pkg.zip](https://github.com/user-attachments/files/18186308/Privileges_2.1.0b1925.pkg.zip) and check if you still see this issue there. If so, please create an issue. Thanks.

**Reply by andrewcrook** [reply]
will do

**Reply by andrewcrook** [reply]
@mthielemann  I can confirm using the beta over a week that the beta does not seem to have this issue. Thanks
